lib/bolt/executor.rb

```diff
@@ -18,14 +18,18 @@ class Executor
     attr_reader :noop, :transports
     attr_accessor :run_as
 
+    # FIXME: There must be a better way
```

Comment:
Probably make them all keyword arguments.

Reply:
Since this requires changes unrelated to this pr in other parts of bolt I figured it'd be a different, maybe maint PR? But I can make those changes here - we only invoke the executor in a handful of places.

Reply:
A separate maint PR, if you feel like addressing it, is fine.